lua/wire/server/wirelib.lua

```diff
@@ -446,6 +446,35 @@ function WireLib.Remove(ent, DontUnList)
 	WireLib._RemoveWire(ent:EntIndex()) -- Remove entity from the list, so it doesn't count as a wire able entity anymore. Very important for IsWire checks!
 end
 
+local UniqueWireID = 0
+local function WireConstraint(Ent1, Ent2)
+	local Constraint = constraint.Find(Ent1, Ent2, "Wire")
+	if Constraint then return Constraint end
+	UniqueWireID = UniqueWireID + 1
+	Constraint = {
```

Comment:
okay, so you're reusing this variable for a different purpose and a different data type.
Can you change that?
Also, almost none of the variable/parameter types you aded matches the naming conventions:
https://github.com/wiremod/wire/wiki/Coding-style#naming